pallets/omnipool/src/lib.rs

```diff
@@ -1286,6 +1301,8 @@ pub mod pallet {
 
 			Self::update_imbalance(state_changes.delta_imbalance)?;
 
+			assert!(asset_in_state.reserve < new_asset_in_state.reserve);
```

Comment:
@enthusiastmartin is this line kept outside of `try-runtime` feature on purpose?

Reply:
should be removed. i think i was testing something and left it there accidentally.